app/controllers/exchanges/hbx_profiles_controller.rb

```diff
@@ -768,23 +790,36 @@ def year_plan_data(year)
   end
 
   def carrier_data(carrier_data, year)
-    carrier_name = carrier_data[0]
-    carrier_ids = carrier_data[1]
+    carrier_name = carrier_data[:legal_name]
+    profile_ids = carrier_data[:profile_ids]
+    organization_id = carrier_data[:organization_id]
     product_query = BenefitMarkets::Products::Product.where(
       :"application_period.min".lte => Date.new(year, 12, 31),
       :"application_period.max".gte => Date.new(year, 1, 1),
-      :issuer_profile_id.in => carrier_ids
+      :issuer_profile_id.in => profile_ids
     )
     product_ids = product_query.pluck(:_id)
     {
       carrier: carrier_name,
+      organization_id: organization_id,
       plans_number: product_query.count,
       pvp_numbers: BenefitMarkets::Products::PremiumValueProduct.where(active_year: year, :product_id.in => product_ids).count,
       enrollments_number: Family.actual_enrollments_number(product_ids: product_ids),
       products: product_query.distinct(:kind).map { |kind| kind.to_s.capitalize }.join(", ")
     }
   end
 
+  def product_data(product)
+    {
+      plan_name: product.title,
+      plan_type: product.kind.to_s.capitalize,
+      pvp_areas: product.premium_value_products.map{ |pvp| pvp.rating_area.human_exchange_provided_code }.join(',').presence || 'N/A',
+      plan_id: product.hios_id,
+      network: 'network',
```

Comment:
Basically speculation, but I wonder if this is meant to be https://github.com/health-connector/enroll/blob/9a2c4713e401a9ef26582451e8f56b736d08578f/components/benefit_markets/app/models/benefit_markets/locations/service_area.rb#L8 for the service_area associated with the product...

Reply:
Sorry, we discussed it with the team but didn't mention it in the ticket.
Ticket description and ticket mock-up have different lat columns.  We decided to go with the "Metal level" from the mock-up.

And about the network. There is a comment in a ticket that we decided to go temp with plug. Caz we are not sure if we need this data, and where to take it.